packages/storybook/stories/user-profile.mdx

```diff
@@ -0,0 +1,35 @@
+import { UserProfile } from '@equisoft/design-elements-react';
+import { ArgTypes, Meta, Canvas } from '@storybook/blocks';
+import * as UserProfileStories  from './user-profile.stories';
+
+<Meta of={UserProfileStories} />
+
+# User Profile
+1. [Definition](#definition)
+2. [Usage](#usage)
+3. [Variants](#variants)
+4. [Properties](#properties)
+
+## Definition
+User profile is a high-level navigation menu regrouping user settings and personal details [1]. 
+<Canvas of={UserProfileStories.Desktop} />
+
+## Usage
+### When to use
+- Use to display information like user profile (name, etc.), login and security (username, password, pin), language and currency settings, appearance (light/dark theme, accessibility, font size), other settings and log out [1]. 
```

Comment:
Est-ce qu'on veut vraiment avoir toute ces informations là directement dans la boite qui expand? On ne risque pas de se ramasser avec un shit load de feature comme dans /Centralize? 
Le User profile ne devrait pas permettre d'accéder à des pages qui permettent de faire ce genre de modification plutôt que de les faire directement dans la section expandable?

Reply:
Ça ne veut pas dire que toutes ces infos là sont là, mais elles peuvent l'être.

Reply:
Oui c'est un peu mon point en fait. Le composant permet d'accéder à des pages comme "My account" mais pas forcément de faire les changements (comme changer ses préférences) directement dans le composant lui même. Le composant permet de naviguer à ces pages mais la façon dont c'est rédiger laisse croire que les modifications se font dans le popup.

Reply:
« Use to display » pas « Use to edit »